pkg/lint/policy_builtins.go

```diff
@@ -435,4 +435,17 @@ var builtinRules = map[string]validators{
 
 		},
 	},
+	"pipeline-has-metadata-push": {
```

Comment:
Is it a good idea to check this on pipeline level ?. I thought enabling metadata was done per run , with `push-metadata` flag ,